qa/rpc-tests/sigma_spend_gettransaction.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+from decimal import *
+
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import *
+
+class SigmaSpendGettransactionTest(BitcoinTestFramework):
+    def __init__(self):
+        super().__init__()
+        self.num_nodes = 4
+        self.setup_clean_chain = True
+
+    def setup_nodes(self):
+        # This test requires mocktime
+        enable_mocktime()
+        return start_nodes(self.num_nodes, self.options.tmpdir)
+
+    def run_test(self):
+        # Decimal formating: 6 digits for balance will be enought 000.000
+        getcontext().prec = 6
+        self.nodes[0].generate(400)
+        self.sync_all()
+
+        # get a watch only address
+        watchonly_address = self.nodes[3].getnewaddress()
+        watchonly_pubkey = self.nodes[3].validateaddress(watchonly_address)["pubkey"]
+        self.nodes[0].importpubkey(watchonly_pubkey, "", True)
+
+        valid_address = self.nodes[0].getnewaddress()
+
+        for _ in range(10):
+            self.nodes[0].mint(1)
+
+        self.nodes[0].generate(10)
+        self.sync_all()
+
+        # case 1: Spend many with watchonly address
+        spendto_wo_id = self.nodes[0].spendmany("", {watchonly_address: 1})
+        spendto_wo_tx = self.nodes[0].gettransaction(spendto_wo_id)
+
+        assert spendto_wo_tx['amount'] == Decimal('-1')
+        assert spendto_wo_tx['fee'] == Decimal('-0.05')
```

Comment:
I think we should not check fee with exactly value here due to this test is not a test for fee algorithm. We should check only if there is fee.